### PR TITLE
[10-10CG] Update test coverage for Form 10-10CG

### PR DESCRIPTION
**Pull request by coope93**
## Summary

- This work is behind a feature toggle (flipper): NO
- Added some specs for logic paths that were not being covered per `simple-cov` gem. These changes get those files from ~98% coverage to 100% coverage. 
- Only rspec tests were updated. No implementation code has changed. 
- Added a test for the private method `log` in the auditor class. It looks like we are not calling it with data that would use the filter value, but I figured it was good to cover it here so I added a test for the private method. If we don't like testing the private method, I think we should remove the logic since we are not currently calling it. 
- 1010 Health Apps

## Testing done

- [x] *New code is covered by unit tests*

## What areas of the site does it impact?
- 10-10CG
## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

File: spec/services/form1010cg/auditor_spec.rb [hunk 1]
```diff
@@ -286,4 +286,25 @@
...
+      {
+        veteran_name: ActiveSupport::ParameterFilter::FILTERED
+      }
+    end
+
+    it 'calls :record_attachments_delivered' do
+      expect(Rails.logger).to receive(:info).with("[#{described_class::LOGGER_PREFIX}] #{message}", **expected_context)
+      
+      described_class.new.send(:log, message, context)
+    end
```
Comment (Mitch-A6) [inline spec/services/form1010cg/auditor_spec.rb:308]: 🤔 not sure what using `let`s offers here? might read clearer in-line?
```suggestion
  describe '#filtering keys' do
    it 'calls :record_attachments_delivered' do
      message = 'my message'
      context = { veteran_name: 'My name' }
      expected_context = { veteran_name: ActiveSupport::ParameterFilter::FILTERED }
      
      expect(Rails.logger).to receive(:info).with("[#{described_class::LOGGER_PREFIX}] #{message}", **expected_context)
      
      described_class.new.send(:log, message, context)
    end
```

**Review by Mitch-A6 — APPROVED** [review]
yay more coverage!

**Review by Mitch-A6 — APPROVED** [review]
👍

**Comment by rmtolmach** [comment]
I updated your branch with master. You had a failing spec that was fixed earlier today in master.

**Review by rmtolmach — APPROVED** [review]
👍 👍

**Pull request merged**